### DISCUSSION TITLE
Bit and vueJS

### DISCUSSION THREAD

**Discussion by stephanfriedrich** [teambit/bit#7231] [Help]
Hi there,
sounds really interesting, is there a plan to make vuejs usable with this ?
couldn't find any current documentation, article or something like that until now.

have a nice day and stays cheerful
Greetings

**Comment by itaymendel** [comment]
https://bit.cloud/teambit/vue
vuejs support is on its way. might take several weeks for us to feel comfortable with the implementation for us to ship something for the community to start looking into.
